### ISSUE TITLE
[Suggestion] Allow disabling of @everyone and individual role mentions in-game

### ISSUE THREAD

**Issue by Caaethil** [Eufranio/MagiBridge#213]
Being able to mention a Discord user via the in-game chat is cool, but being able to @everyone is a big problem. Regular users are typically not able to @everyone in Discord servers due to how it can be abused, but anyone can do it via MagiBridge.

I'm also going to suggest allowing us to disable mentions on a per-role basis, though role pings don't seem to be working right now anyway (I opened another issue regarding this).

**Comment by Mike5357** [comment]
Set up your bot permissions so that it is not an Administrator, and does not have permission for @everyone

**Comment by Caaethil** [comment]
Pretty sure I tried that before I posted this issue (I should have mentioned as much), but I tried it again now, just to be safe. My bot has two roles (a role belonging to all bots and its own role). Neither are administrator roles and neither have the "Mention @ everyone, @ here, and All Roles" permission. I also explicitly denied this permission in all of the channels in my server that the bot posts to (for both of those roles). There are no permissions assigned to the bot *user* in these channels that would override the permissions set for these roles.

I'm wondering if maybe the webhooks themselves have different permissions than the bot (though this doesn't really make any sense to me). I also considered that maybe this was a permissions thing I was supposed to have configured when setting up the bot, but I'm pretty sure that anything a user assigns to a bot after the fact via roles should override anything the bot says it needs when joining a server.

Thanks for the response anyway - this may well still be something dumb I've set up wrong as opposed to an actual issue, but I can't find anything.
